tldraw/tldraw
# change pointer cursor to white

This PR changes the pointer cursor back to white (from black).

I made it black to do some debugging, and I forgot to change it back.

### Change Type


### Test Plan

1. Make a bookmark shape by pasting a link.
2. Hover the hyperlink of the bookmark.
3. Check that the 'pointer' cursor is white.
